### ISSUE TITLE
windows build

### ISSUE THREAD

**Issue by sunwangshu** [genekogan/KinectOSC#3]
It seems the current build only has mac version. Would you like to add the windows build as well? Thanks!

**Comment by genekogan** [comment]
yes, this should be able to compile for windows if you are able to get ofxOpenNI working correctly (I'm not sure what its status is).

**Comment by sunwangshu** [comment]
Thanks, I tried VS studio and openframeworks but all the required plugins later on sorta intimidates me...
